src/webserver_main.cpp

```diff
@@ -140,6 +140,9 @@ static int run()
     crow::dbus_monitor::registerBIOSAttrUpdateSignal();
     // Start event log entry created monitor
     crow::dbus_monitor::registerEventLogCreatedSignal();
+    // Start hypervisor app dbus monitor for hypervisor
+    // network configurations
+    crow::dbus_monitor::registerVMIConfigChangeSignal();
```

Comment:
Had this here for 1030 
https://github.com/ibm-openbmc/bmcweb/blob/2c5b246b9aea8f51263d21e3f73747b181f6ccd5/src/webserver_main.cpp#L177